CONTRIBUTING.md

```diff
@@ -37,6 +37,10 @@ However, you do not frequently have to build all of kubernetes to work on kubead
 
 See [./vagrant/README.md](./vagrant/README.md) for a quick workflow to build and test your own kubeadm binaries. 🙂
 
+### Testing pre-release versions of Kubernetes with kubeadm
+
+See [testing-pre-releases.md](testing-pre-releases.md) for information about how to get pre-release versions of kubernetes or kubeadm and how to test it.
```

Comment:
`s/test it/test them/`

Reply:
There's plenty of information on how to get these prerelease versions, but not much on testing. What would a deployment look like? How can you test say, a pre-release kubelet binary? What verification steps should be run?